packages/genui/a2ui-playground/src/styles.css

```
@@ -1000,6 +1025,36 @@ a {
+
+.chatCodeBlockPre {
+  margin: 0;
+  padding: 12px;
+  overflow-x: auto;
+  background: var(--geist-code-bg);
+  color: var(--geist-code-fg);
+  font-family: var(--geist-mono);
+  font-size: 12px;
+  line-height: 1.6;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)